content/en/docs/instrumentation/cpp/exporters.md

```diff
@@ -82,6 +82,21 @@ auto zipkin_exporter =
 
 ```
 
+## Prometheus
+
+To send metrics to a prometheus endpoint you'll want to configure a prometheus
+exporter
+
+```cpp
+opentelemetry::exporter::metrics::PrometheusExporterOptions opts;
+opts.url = "localhost:8080";
+std::unique_ptr<opentelemetry::sdk::metrics::MetricExporter> exporter{new opentelemetry::exporter::metrics::PrometheusExporter(opts)};
+std::unique_ptr<opentelemetry::sdk::metrics::MetricReader> reader{
+    new opentelemetry::sdk::metrics::PeriodicExportingMetricReader(std::move(exporter), options)};
+```
```

Comment:
To make it look complete, we can define `options` at the beginning.

```cpp
opentelemetry::sdk::metrics::PeriodicExportingMetricReaderOptions options;
options.export_interval_millis = std::chrono::milliseconds(1000); //optional, to override default values
options.export_timeout_millis  = std::chrono::milliseconds(500); // optional, to override default values
```

Reply:
thanks, I updated it accordingly.